src/functions_framework/__init__.py

```diff
@@ -288,8 +349,9 @@ def handle_none(rv):
     # Get the configured function signature type
     signature_type = _function_registry.get_func_signature_type(target, signature_type)
     function = _function_registry.get_user_function(source, source_module, target)
+    inputType = _function_registry.get_func_input_type(target)
```

Comment:
Hmm, maybe controversial, but given that a minority of function types will use the inputType parameter, I think we should actually do this lookup in the if condition of the _configure_app function instead of here.

Reply:
Good point. I will make the changes